### ISSUE TITLE
Some problems in changing teams and dates

### ISSUE THREAD

**Issue by Sakr9** [JuliusRWash/NBA-Game-Prediction#1]
Hello and thank you for your work. I ran into some problems, especially in this part of the code:

`warriors_id = rankings[rankings['TEAM'] == 'Golden State']['TEAM_ID'].iloc[0]
celtics_id = rankings[rankings['TEAM'] == 'Boston']['TEAM_ID'].iloc[0]
print(f'Warriors id: {warriors_id}, Celtics id: {celtics_id}')
date = '2022-12-10'
game = games[(games['GAME_DATE_EST'] == date) & (games['HOME_TEAM_ID'] == warriors_id) & (games['VISITOR_TEAM_ID'] == celtics_id)]
game`

If I try to change the date, for example to simulate today's game, the table is empty. I also tried entering December 9th or December 11th as dates but the result is the same, it would seem to work only with December 10th.

This also blocks the next piece of code because it obviously fails to retrieve any data.

```
warriors_win = game['HOME_TEAM_WINS'].iloc[0]
warriors_win
```

Finally one last piece of information, how to interpret the values that come out at the end of the whole calculation? I see all 100 EPOCHs with the four values (Train/Val loss, Train/Val acc) but nothing more. It would be useful to have a clearer output that inserts the two teams, the date and a possible percentage for the single team (if any) or in any case clearly the winning team. Thanks and nice work!
